packages/elements/src/slider/index.ts

```diff
@@ -88,12 +48,95 @@ export class Slider extends ControlElement {
     return VERSION;
   }
 
-  private activeThumb!: HTMLElement;
+  /**
+   * Define styles in a tagged template literal, using the css tag function.
+   * @returns CSS template
+   */
+  static get styles (): CSSResultGroup {
```

Comment:
This method should be moved to the sibling `render` method. Ref: https://confluence.refinitiv.com/display/ELF/Code+Styling+Guide

Reply:
It should not. static methods come at the top. We need to change the guidelines